test/dummy/app/controllers/comboboxes_controller.rb

```diff
@@ -51,4 +51,9 @@ def render_in
     def set_states
       @states = combobox_options State.all, id: :abbreviation, value: :abbreviation, display: :name
     end
+
+    def variations
```

Comment:
I'm not thrilled about making assumptions about a specific variable name being tied to a side nav. What do you think about this solution of letting each controller define whether a side nav exists (this could even be action-specific) and only assume the name of the `_nav` partial?

Reply:
That works!